package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "mcp-storage-server",
+  "version": "1.0.0",
+  "description": "Storacha MCP Storage Server - Decentralized storage for AI systems",
+  "type": "module",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc",
+    "start": "node dist/index.js",
+    "start:stdio": "MCP_TRANSPORT_MODE=stdio node dist/index.js",
+    "start:sse": "MCP_TRANSPORT_MODE=sse node dist/index.js",
```

```suggestion
    "start:stdio": "MCP_TRANSPORT_MODE=stdio pnpm start",
    "start:sse": "MCP_TRANSPORT_MODE=sse pnpm start",
```

...you might have to check these env vars are persisted but ideally just call start so that if the start script changes we don't accidentally forget to update it in the other places.